crates/optimism/trie/src/api.rs

```diff
@@ -57,6 +57,14 @@ pub struct BlockStateDiff {
     pub post_state: HashedPostState,
 }
 
+impl BlockStateDiff {
+    /// Extend the `BlockStateDiff` from other latest `BlockStateDiff`
```

Comment:
```suggestion
    /// Extend the [` BlockStateDiff`] from other latest [`BlockStateDiff`]
```
nitpick, allows compiler to help us catch type name changes